backend/src/packages/auth/auth.service.ts

```diff
@@ -187,6 +187,20 @@ class AuthService {
     };
   }
 
+  public async getCurrent(
```

Comment:
As for now this looks more like a "attachUserBusiness" method. Although, I understand why you're doing like this.

I believe, that in the future all getUserBy... methods of userService will be returning user with optional business attached, thus AuthStrategy.INJECT_USER also would have it returning automatically.